src/windows.rs

```diff
@@ -35,15 +41,33 @@ pub fn getrandom_inner(dest: &mut [MaybeUninit<u8>]) -> Result<(), Error> {
         };
         // NTSTATUS codes use the two highest bits for severity status.
         if ret >> 30 == 0b11 {
-            // We zeroize the highest bit, so the error code will reside
-            // inside the range designated for OS codes.
-            let code = ret ^ (1 << 31);
-            // SAFETY: the second highest bit is always equal to one,
-            // so it's impossible to get zero. Unfortunately the type
-            // system does not have a way to express this yet.
-            let code = unsafe { NonZeroU32::new_unchecked(code) };
-            return Err(Error::from(code));
+            // Failed. Try RtlGenRandom as a fallback.
+            let success = fallback_rng(chunk);
```

Comment:
This introduces a larger diff and more indentation than needed. Could we just do:

```rust
// NTSTATUS codes use the two highest bits for severity status.
if ret >> 30 == 0b11 {
    // Failed. Try RtlGenRandom as a fallback.
    #[cfg(not(target_vendor = "uwp"))]
    if unsafe { RtlGenRandom(chunk.as_mut_ptr() as *mut c_void, chunk.len() as u32) } != 0 {
        continue;
    }

    // We zeroize the highest bit, so the error code will reside
    // inside the range designated for OS codes.
    let code = ret ^ (1 << 31);
    ...
```